go.mod

```diff
@@ -60,9 +60,19 @@ require (
 )
 
 require (
+	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/cespare/xxhash/v2 v2.1.2 // indirect
 	github.com/containerd/cgroups/v3 v3.0.0-20221112182753-e8802a182774 // indirect
+	github.com/docker/go-metrics v0.0.1 // indirect
+	github.com/matttproud/golang_protobuf_extensions v1.0.4 // indirect
+	github.com/prometheus/client_golang v1.14.0 // indirect
+	github.com/prometheus/client_model v0.3.0 // indirect
+	github.com/prometheus/common v0.37.0 // indirect
+	github.com/prometheus/procfs v0.8.0 // indirect
+	github.com/tchap/go-patricia/v2 v2.3.1 // indirect
 	golang.org/x/oauth2 v0.0.0-20221014153046-6fdb5e3db783 // indirect
 	google.golang.org/genproto v0.0.0-20221206210731-b1a01be3a5f6 // indirect
+	k8s.io/cri-api v0.26.0-beta.0 // indirect
```

Comment:
Can we drop this dependency by copying `containerd/pkg/cri/store/container/metadata.go`?

Reply:
yeah，we can drop this if we copy `containerd/pkg/cri/store/container/metadata.go`